examples/02_model_collaborative_filtering/standard_vae_deep_dive_2.ipynb

```diff
@@ -0,0 +1,2268 @@
+{
```

Comment:
<p>could you please erase these logs? also, please note that the google colab info is still on this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
ok I will also change this file too.

Reply:
yes sure ! I will do it first thing in the morning.